geode-dunit/src/main/java/org/apache/geode/management/internal/configuration/ClusterConfig.java

```diff
@@ -57,6 +59,27 @@ public ClusterConfig(ConfigGroup... configGroups) {
     Collections.addAll(this.groups, configGroups);
   }
 
+  private static Set<String> toSetIgnoringHiddenFiles(String[] array) {
+    if (array == null) {
+      return new HashSet<>();
```

Comment:
If callers cannot modify the returned Set, then you might want to return Collections.emptySet() instead.